### DISCUSSION TITLE
11 release date

### DISCUSSION THREAD

**Discussion by chris835** [AvaloniaUI/Avalonia#9648] [General]
Looking at [this](https://github.com/AvaloniaUI/Avalonia/projects/10) it seems that 11 is getting pretty close. Any estimates on when it'll be released?

**Comment by kekekeks** [comment]
Most likely Q1 2023

**Reply by chris835** [reply]
Looking forward to it!

**Reply by timunie** [reply]
Some folks are already on the preview releases or even use nightly builds. If you feel confident enough to do, you can also update to it. Any issue you find and report helps us to make 11.0 as stable as possible.

**Reply by mozesa** [reply]
Will Citrus.Avalonia be updated to support the upcoming v11?

**Reply by timunie** [reply]
I think you need to ask them directly. We do not maintain this third party package as far as I know.

**Reply by kuiperzone** [reply]
Great! Can't wait!

Do we have anything on the roadmap for a version 1.0.0? Is that years away, or sooner?

**Reply by timunie** [reply]
I think there will never be a version 1. Internally we see 0.10 as stable as a v1.0. so the decision was to make the next release 11.0 instead of 0.11

I hope that makes sense

**Reply by kuiperzone** [reply]
Edited. Yes, understand. Version 11.0! Great! :)

**Reply by maxkatz6** [reply]
@timunie @mozesa Citrus is currently under Avalonia organization on github. It will be updated to 11.0, but probably after RC which is planned next 1-2 months.

**Reply by mozesa** [reply]
@maxkatz6 Okay, that's what I hoped. 🙂
Thanks for the answear.

**Comment by cesarchefinho** [comment]
any updates about avalonia rc release date?
